sqlglotrs/src/tokenizer.rs

```diff
@@ -395,6 +395,10 @@ impl<'a> TokenizerState<'a> {
                 .push(self.text()[comment_start_size..].to_string());
         }
 
+        if comment_start == self.settings.hint_start && self.tokens.last().is_some() && self.settings.tokens_preceding_hint.contains(&self.tokens.last().unwrap().token_type) {
```

Comment:
Let's manually format this a bit, I think the pattern is to have && in the next line.